package.json

```diff
@@ -37,6 +37,8 @@
     "lib/processor.js"
   ],
   "devDependencies": {
+    "@types/eslint": "^7.2.7",
+    "@types/unist": "^2.0.3",
```

Comment:
ESLint projects use JSDoc types without adopting TypeScript/Flow/etc-specific semantics, so I'd like to avoid installing TypeScript declarations from `@types/*`. If that requires re-defining JSDoc `@typedef`s for the `ASTNode` and `Message` types, that duplication is not a problem.

```suggestion
```